### ISSUE TITLE
Bad layout in landscape

### ISSUE THREAD

**Issue by Jean-BaptisteC** [meenbeese/Chronos#24]
## Device Info

- Brand: `Google`
- Model name: `Pixel 6`
- Android version: `14`

## Steps to Reproduce

- start app
- rotate device

<img src="https://github.com/meenbeese/Chronos/assets/87148630/526c537c-fe09-4e13-853b-00143827c49e" height=200 />

**Comment by Saipavan1239** [comment]
I think i can help you with tht but i need a bit of details of how u want it to be done (like how do u want me to make it).

**Comment by meenbeese** [comment]
Thank you for your interest @Saipavan1239.

A good starting point would be to center the image properly and show the `You do not have any alarms...` text properly, which is currently cut off entirely. Also, the clock on the top is huge in landscape, which should also be fixed. I think the UI when the have at least one alarm is manageable for now. A long term plan could be to move the tabs to the left side on landscape (not sure if it is even doable) or maybe find some other creative solution to replace the tabs?

**Comment by Saipavan1239** [comment]
![image](https://github.com/meenbeese/Chronos/assets/156532921/6224007e-6a8e-4dcf-8a47-b7d59ac5018f)
how does this look or do u need any modifications.
If u want to do any changes i'll do it and will add the xml code

**Comment by meenbeese** [comment]
Yes, this looks good to me. It is exactly how I imagined an optimal solution would look like. Maybe the alarm sizes can be adjusted so that they don't span the whole width, or more alarms can be displayed next to each other, but that can be discussed later on. For now, this is a great start.

**Comment by Saipavan1239** [comment]
so how can i send u the code for this design i just made this design as simpler as possible, maybe i can add the adjustments i want so how can we take this forward should i just complete the code and send u some link or do you have anyother idea.

**Comment by meenbeese** [comment]
Please raise a PR (using Git) against the main branch of this repo so that we can review and discuss your changes. Also, make sure to include screenshots of your changes.

**Comment by Saipavan1239** [comment]
I would certainly do that, but the problem is that I still need to make some changes to the code and it will keep me busy for a week or two, after which will I be able to raise the PR and get the code fully finished and submit it?

**Comment by meenbeese** [comment]
Yes, take your time to write well-polished code instead of rushing it and doing a half-baked PR.
